### DISCUSSION TITLE
Default error messages should change.

### DISCUSSION THREAD

**Discussion by harshalkumar-ishi** [pydantic/pydantic#7425] [Feature Request]
Default error messages like as follows:

username
  String should match pattern '^[a-z0-9_-]+$' [type=string_pattern_mismatch, input_value='dfgasdg%', input_type=str]
    For further information visit https://errors.pydantic.dev/2.3/v/string_pattern_mismatch

name
  String should have at most 10 characters [type=string_too_long, input_value='My Name Is Harshal', input_type=str]
    For further information visit https://errors.pydantic.dev/2.3/v/string_too_long

There is no option to change those specific messages.
It should have a option to change the messages in model.

**Comment by HansBambel** [comment]
I also find it a bit overkill to add a link for every validation error. The links should be removable as the error description is quite explicit already.

**Comment by warvariuc** [comment]
`pydantic.ValidationError.errors` method has `include_url` argument:
```python
exc.errors(include_url=False)
```

**Reply by HansBambel** [reply]
If this disables the `For further information`-bit, is there a way to enable this by default?
